_locales/_untranslated_en.json

```diff
@@ -34,5 +34,17 @@
   },
   "switch_account_desktop":{
     "message": "Switch account"
+  },
+  "select_profile_image_desktop": {
```

Comment:
Please remove “desktop” prefix. Other clients will use this.

Reply:
this is desktop specific, its the title of the file picker window.

Reply:
hm, in general i agree with @OzancanKaratas , we should create os-specific strings only if it happens that we really need them. for most other hints etc. we do not have specific stuff.
This "Select your new profile image" might be useful on other os as well.
however, we can streamline this when adding it to androids strings.xml, this should not block merging.